build/Jenkinsfile

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env groovy
 
 pipeline {
+
+    options { buildDiscarder(logRotator(numToKeepStr: '2')) }
```

Comment:
What's the impact of the value of `2`? Can we only see results from the 2 most recent Jenkins builds? What about lots of active PRs with builds in various stages?

Reply:
yes, only the last two builds, usually you will see the current build and the previous one. But I will increase this to 5.

Reply:
And just to confirm, is this `5` per PR, or `5` overall across all active PRs?

Reply:
5 build history per active PR